src/Internal/Service/ContextLoggerService.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Class ContextLoggerService
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Service;
+
+use WCPay\Internal\Logger;
+use WCPay\Internal\Payment\PaymentContext;
+use WCPay\Internal\Payment\Change;
+
+/**
+ * Service for logging payment context.
+ */
+class ContextLoggerService {
```

Comment:
I'd be ok with putting helpers like this within the `Payment` namespace, but if we're putting them in the global `Services` one, let's add "payment" to the class name. So this would either be `WCPay\Internal\Service\PaymentContextLoggerService` or `WCPay\Internal\Payment\ContextLogger`. I'd favor the latter.

Reply:
Done